### PR TITLE
Feature/e2 e 1662

### PR DESCRIPTION
**Pull request by ghost**
* Add stories for Lazy, Spinner and Grid

### CODE REVIEW

File: src/assests/skeltons/ListingSkelton.component.tsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import {
  Skeleton, Layout, PageHeader, Row, Col,
 } from 'antd';
 import React, { FC, memo } from 'react';
-import TableSkelton from 'assests/skeltons/TableSkelton.component';
```
Comment (ghost) [inline src/assests/skeltons/ListingSkelton.component.tsx:5]: relative imports are not working in storybook

Comment (rachit-aasaanjobs) [reply src/assests/skeltons/ListingSkelton.component.tsx:5]: you have fixed imports now, right ?

**Comment by rachit-aasaanjobs** [comment]
@ameerjhan there are conflicts, please resolve them

**Comment by ghost** [comment]
> @ameerjhan there are conflicts, please resolve them

fixed the merge conflicts

File: src/assests/skeltons/ListingSkelton.component.tsx (hunk 1 above)
Comment (ghost) [reply src/assests/skeltons/ListingSkelton.component.tsx:5]: yes

**Review by rachit-aasaanjobs — APPROVED** [review]

**Pull request merged**